probes/hasOSVVulnerabilities/impl.go

```diff
@@ -65,10 +68,12 @@ func Run(raw *checker.RawResults) ([]finding.Finding, string, error) {
 		if err != nil {
 			return nil, Probe, fmt.Errorf("create finding: %w", err)
 		}
-		f = f.WithMessage(fmt.Sprintf("Project is vulnerable to: %s",
-			strings.Join(vuln.IDs, " / ")))
+		f = f.WithMessage("Project is vulnerable to: " + strings.Join(vuln.IDs, " / "))
+		if len(vuln.IDs) == 0 {
+			return nil, Probe, errNoVulnID
+		}
```

Comment:
```suggestion
		if len(vuln.IDs) == 0 {
			return nil, Probe, errNoVulnID
		}
		f = f.WithMessage("Project is vulnerable to: " + strings.Join(vuln.IDs, " / "))
```

Since `f` won't be used if we enter the `if` block.

Reply:
Done in 60c806e3deeada7b970701dffc6cc8828b6bb955, except I moved it slightly higher so we don't create `f` at all.